templates/zerver/integrations/errbot.md

```diff
@@ -25,23 +27,42 @@ Run your favorite chatbot in Zulip!
         BOT_LOG_FILE = r'<path/to/your/errbot/logfile.log>'
         BOT_LOG_LEVEL = logging.INFO
 
-        BOT_IDENTITY = {  # Fill this with the corresponding values in your bot's `.zuliprc`
-          'email': '<err-bot@your.zulip.server>',
+        BOT_IDENTITY = {
+          'email': '<your-bot-name@yourZulipDomain.zulipchat.com>',
           'key': '<abcdefghijklmnopqrstuvwxyz123456>',
-          'site': '<http://your.zulip.server>'
+          'site': '<https://yourZulipDomain.zulipchat.com>'
```

Comment:
Switch to consistently using "yourZulipDomain" because that's what we're using in our [context](https://github.com/zulip/zulip/blame/main/zerver/views/documentation.py/#L53) across all docs.